spec/rubocop/cop/style/redundant_line_continuation_spec.rb

```diff
@@ -134,6 +134,20 @@ def self.foo(bar,#{trailing_whitespace}
     RUBY
   end
 
+  it 'does not register an offense when line continuations for double quoted string' do
+    expect_no_offenses(<<~'RUBY')
+      "foo \
+        bar"
```

Comment:
The following code causes the spec to fail.
```suggestion
      a = "foo \
            bar"
```

Reply:
Oops, I overlooked this case. I've fixed it and updated this PR.